src/main/frontend/src/components/AddCatalogPriceForm.js

```diff
@@ -2,12 +2,162 @@
 // IMPORTANT:
 // You must update ResourceUrl.RESOURCES_VERSION each time whenever you're modified this file!
 //
+// @todo #1342 AddCatalogPriceForm: show a currency
 
 class AddCatalogPriceForm extends React.Component {
-	
+	constructor(props) {
+		super(props);
+		this.state = {
+			price: null,
+			catalog: 'michel',
+			hasServerError: false,
+			validationErrors: [],
+			isDisabled: false,
```

Comment:
We should append trailing comma everywhere or remove it from everywhere. Anyway, the code style should be consistent.
```suggestion
			isDisabled: false
```